edx_exams/apps/router/interop.py

```diff
@@ -0,0 +1,21 @@
+"""
+Module for writing data to edx-proctoring
+"""
+from posixpath import join as urljoin
+
+from django.conf import settings
+from requests.exceptions import HTTPError
+
+from edx_exams.apps.core.rest_utils import get_client, make_request
+
+LMS_REGISTER_PROCTORED_EXAMS_API_TPL = 'api/edx_proctoring/v1/proctored_exam/exam_registration/course_id/{}'
+
+def register_exams(course_id, request_data):
```

Comment:
request data is a list of exams, maybe call it exams or exam_list or something

or just quickly go add types to everything in edx-exams ;)